log-viewer/modules/TreeView.ts

```diff
@@ -492,6 +502,65 @@ function hideByDuration(elements: Array<HTMLElement>) {
   }
 }
 
+// Find the parent node of "elm" (or return null if we reach the root of the tree)
+function getParentNode(elm: Element | null) {
+  if (elm === null || elm.classList.contains('root')) {
+    return null;
+  }
+
+  let parent = elm.parentElement;
+  while (parent && !parent.classList.contains('node')) {
+    parent = parent.parentElement;
+  }
+
+  return parent;
+}
+
+function insertCrumb(container: Element, node: HTMLElement) {
+  // @ts-ignore (custom dom property)
```

Comment:
You should be able to modify the global scope like this.
Then typescript will understand the line property on `HTMLElement`
``
declare global {
  interface HTMLElement {
    line: LogLine;
  }
}
```

https://www.typescriptlang.org/docs/handbook/declaration-merging.html#global-augmentation

Reply:
Done.